gateway/gateway.go

```diff
@@ -917,34 +945,24 @@ func (gateway *HandleT) setWebPayload(r *http.Request, qp url.Values, reqType st
 }
 
 func (gateway *HandleT) pixelHandler(w http.ResponseWriter, r *http.Request, reqType string) {
-	if r.Method == http.MethodGet {
-		queryParams := r.URL.Query()
-		if writeKey, present := queryParams["writeKey"]; present && writeKey[0] != "" {
-			// make a new request
-			req, _ := http.NewRequest(http.MethodPost, "", nil)
 
-			// set basic auth header
-			req.SetBasicAuth(writeKey[0], "")
-			delete(queryParams, "writeKey")
+	queryParams := r.URL.Query()
+	writeKey := queryParams["writeKey"]
+	// make a new request
+	req, _ := http.NewRequest(http.MethodPost, "", nil)
 
-			// set X-Forwarded-For header
-			req.Header.Add("X-Forwarded-For", r.Header.Get("X-Forwarded-For"))
+	// set basic auth header
+	req.SetBasicAuth(writeKey[0], "")
+	delete(queryParams, "writeKey")
 
-			// convert the pixel request(r) to a web request(req)
-			err := gateway.setWebPayload(req, queryParams, reqType)
-			if err != nil {
-				http.Error(w, err.Error(), http.StatusBadRequest)
-				return
-			}
+	// set X-Forwarded-For header
+	req.Header.Add("X-Forwarded-For", r.Header.Get("X-Forwarded-For"))
+
+	// convert the pixel request(r) to a web request(req)
+	_ = gateway.setWebPayload(req, queryParams, reqType)
```

Comment:
handle error here.

Reply:
error is ignored because we are ideally trying to send 200 with pixel value irrespective of any situation